### PR TITLE
Ci/lb bugfixes windows

### PR DESCRIPTION
**Pull request by macintoshpie**
#### What does this PR accomplish?
Add windows OS testing env to CI
#### What are the relevant tickets?
#355

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -9,15 +9,12 @@ isolated_build=True
 [testenv:python]
 commands=
     poetry install --no-root -v
-    poetry run pytest --doctest-modules -v --cov-report term-missing --cov .
-    poetry run coveralls
```
Comment (macintoshpie) [inline tox.ini:13]: no need to run coveralls when running tests locally

File: tests/system_parameters/test_system_parameters.py [hunk 2]
```diff
@@ -64,15 +64,15 @@ def test_expanded_paths(self):
         for s in sdp.validate():
             print(s)
         value = sdp.get_param_by_building_id("ijk678", "load_model_parameters.spawn.idf_filename")
-        self.assertEqual(value, os.path.join(os.path.dirname(filename), 'example_model.idf'))
+        self.assertEqual(Path(value), Path(os.path.dirname(filename)) / 'example_model.idf')
```
Comment (macintoshpie) [inline tests/system_parameters/test_system_parameters.py:67]: windows was failing here due to slash conventions

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -36,25 +41,51 @@ jobs:
...
         name: Tox ${{ matrix.test_env }}
         env:
-          MODELICAPATH: /home/runner/work/modelica-buildings
           COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
-        run: poetry run tox -e ${{ matrix.test_env }}
+        run: |
+          POSARGS=""
+          if [ '${{ matrix.test_env }}' == 'python' ] && [ '${{ matrix.os }}' == 'windows-latest' ]; then
+            # for windows, skip python tests that require simulation (currently broken)
+            POSARGS="-- -m 'not simulation' ./tests"
```
Comment (macintoshpie) [inline .github/workflows/ci.yml:79]: skip any simulation tests for now on windows bc jmodelica script doesn't want to work

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -36,25 +41,51 @@ jobs:
...
-        name: Install dependencies
-        env:
-          MODELICAPATH: /home/runner/work/modelica-buildings
+        name: Install modelicafmt
         run: |
-          # Install modelicafmt (hard code the version for now)
-          curl -SLO https://github.com/urbanopt/modelica-fmt/releases/download/v0.2-pr.1/modelica-fmt_0.2-pr.1_Linux_x86_64.tar.gz
-          tar xzf modelica-fmt_0.2-pr.1_Linux_x86_64.tar.gz
+          RUNNER_SYSTEM=$(python -c 'import platform; print(platform.system())')
+          curl -SLO "https://github.com/urbanopt/modelica-fmt/releases/download/v0.2-pr.1/modelica-fmt_0.2-pr.1_${RUNNER_SYSTEM}_x86_64.tar.gz"
```
Comment (vtnate) [inline .github/workflows/ci.yml:47]:
```suggestion
          curl -SLO "https://github.com/urbanopt/modelica-fmt/releases/download/v0.2-pr.2/modelica-fmt_0.2-pr.2_${RUNNER_SYSTEM}_x86_64.tar.gz"
```

File: .github/workflows/ci.yml [hunk 5]
```diff
@@ -36,25 +41,51 @@ jobs:
...
+          tar xzf modelica-fmt_0.2-pr.1_${RUNNER_SYSTEM}_x86_64.tar.gz
```
Comment (vtnate) [inline .github/workflows/ci.yml:48]:
```suggestion
          tar xzf modelica-fmt_0.2-pr.2_${RUNNER_SYSTEM}_x86_64.tar.gz
```

File: tests/system_parameters/test_system_parameters.py (hunk 2 above)
Comment (vtnate) [reply tests/system_parameters/test_system_parameters.py:67]:
```suggestion
        self.assertEqual(Path(value), Path(filename).parent / 'example_model.idf')
```
```suggestion
        self.assertEqual(Path(value), Path(filename).parent / 'example_model.idf')
```

File: .github/workflows/ci.yml (hunk 3 above)
Comment (vtnate) [reply .github/workflows/ci.yml:79]: We'll need to get that working eventually.

**Review by vtnate — CHANGES_REQUESTED** [review]
Looks great. Can we use the new version of modelica format?

**Pull request merged**